game_starter.py

```diff
@@ -111,20 +158,55 @@ def game_loop(secret_word):
 
     Follows the other limitations detailed in the problem write-up.
     '''
-    # FILL IN YOUR CODE HERE...
-    pass
-
-
 
+    guesses_remaining = 8
+    letters_guessed = []
+
+    print ('Let the game begin!')
+    print('I am thinking of a word with', secretWordLength(secret_word), 'letters.')
+
+    while is_word_guessed(secret_word, letters_guessed) == False:
+      print('You have ', guesses_remaining, 'guesses remaining.')
+      print('Letters available to you: ', get_available_letters(letters_guessed))
+      guessed_input = input('Guess a Letter: ')
+
+      if guessed_input in get_available_letters(letters_guessed):
+        letters_guessed.append(guessed_input)
+        if(guessed_input in secret_word):
+          print('Correct: ', get_guessed_word(secret_word, letters_guessed))
+        else:
+          print('Incorrect, this letter is not in my word: ', get_guessed_word(secret_word, letters_guessed))
+          guesses_remaining -= 1
+      else:
+        print('You fool! You tried this letter already: ', get_guessed_word(secret_word, letters_guessed))
+
+      if guesses_remaining == 0:
+        break
+
+    if is_word_guessed(secret_word, letters_guessed) == True:
```

Comment:
You alread check is_word_guessed as the while loop condition.
Why check is_word_guessed again ?
Should be simplified.  @Kanyanat-Grace